test/helpers/assert.mligo

```diff
@@ -0,0 +1,60 @@
+#import "../../src/anti.mligo" "ANTI"
+#import "../functions.mligo" "FUNC"
+
+(* Assert contract result is successful *)
+let tx_success (res: test_exec_result) : unit =
+    match res with
+        | Fail (Rejected (error,_)) -> let () = Test.log(error) in failwith "Transaction should not fail"
+        | Fail _ -> failwith "Transaction should not fail"
+        | Success(_) -> Test.log("OK", res)
+
+(* Assert contract call results in failwith with given string *)
+let string_failure (res : test_exec_result) (expected : string) : unit =
+    let _expected = Test.eval expected in
+    let () = match res with
+        | Fail (Rejected (actual,_)) -> assert (actual = _expected)
+        | Fail (Balance_too_low err) -> failwith "Contract failed: Balance too low"
+        | Fail (Other s) -> failwith s
+        | Success _ -> failwith "Transaction should fail"
+    in
+    Test.log("OK", expected)
+
+let assert_transfer_account(ant_addr, sender, recipient, init_token_balance, tsfr_amount : (ANTI.parameter, ANTI.storage) typed_address * address * address * nat * nat) : unit =
```

Comment:
I think it should be more generic : assert_balance(taddr, address, amount)
An assert function should not contain the values it asserts, they must be passed as param

Reply:
The current params are (contract, sender, recipient, before_amount, transfer_amount)
The only coded values inside the Assert are the Burn/Reserve percentages
What values are not good for you ?

Reply:
Sorry, I though the values were something else, I guess the problem is in the contract, `1` and `7` should be constants